### ISSUE TITLE
Problem with StaticEngine.PATTERN_KEY_VALUE

### ISSUE THREAD

**Issue by Thieumdesp** [wuic/wuic#204]
The pattern uses \r\n. On unix/linux system endofline is \n (_without \r_).
**wuic-static files** generated on :
- windows have \r\n as endofline => fine with the pattern
- unix have \n only => not fine with the pattern

a solution could be to remove \r from the pattern :
- BEFORE :

``` java
public static final Pattern PATTERN_KEY_VALUE = Pattern.compile("(.+)\\s\"*((?<=\")[^\"]+(?=\")|([^\\r\\n]+))\"*");
```
- AFTER :

``` java
public static final Pattern PATTERN_KEY_VALUE = Pattern.compile("(.+)\\s\"*((?<=\")[^\"]+(?=\")|([^\\n]+))\"*");
```

**Comment by gdrouet** [comment]
PR is welcome! You should use `System.getProperty("line.separator")` instead. Thanks!

**Comment by gdrouet** [comment]
Fixed by commit d9617ed4d0137ea6a2cf606911dda05e12100b00
